### PR TITLE
feat(hpc-vmware-managed-vcd): add options tile license

### PR DESCRIPTION
**Pull request by pauldkn**
ref: MANAGER-14872

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/hpc-vcd-beta`
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | feat/MANAGER-14872
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-options-tile/OrganizationOptionsTile.component.tsx [hunk 1]
```diff
@@ -12,9 +12,7 @@ export default function OrganizationOptionsTile() {
       <div className="flex flex-col w-full">
```
Comment (chipp972) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-options-tile/OrganizationOptionsTile.component.tsx:12]: There is a component in the lib for this : DashboardTile
You should maybe use it to not have to update it later with the ODS versions

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-options-tile/OptionsTileLicense.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import useManagedVcdOrganization from '@/data/hooks/useManagedVcdOrganization';
+import TileSubtitle from '../tile-subtitle/TileSubtitle.component';
+
+export default function OptionsTileLicense() {
+  const { t } = useTranslation('dashboard');
+  const { id } = useParams();
+  const { data: vcdOrganization } = useManagedVcdOrganization(id);
```
Comment (rjamet-ovh) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-options-tile/OptionsTileLicense.tsx:11]: Loading and error state management are missing

**Review by chipp972 — APPROVED** [review]

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/billing-tile/BillingTile.component.tsx [hunk 3]
```diff
@@ -26,40 +24,48 @@ export default function BillingTile({ id }: TBillingTileProps) {
...
-          </OsdsChip>
-        </div>
-      </div>
-    </OsdsTile>
+    <div className="h-fit">
+      <DashboardTile
+        title={t('managed_vcd_dashboard_service_management')}
+        items={[
+          {
+            id: '',
```
Comment (darsene) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/billing-tile/BillingTile.component.tsx]: Why id is always `''` ? I guess it's used for key prop so it should be set with an unique value, right ?

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-general-information-tile/OrganizationGeneralInformationTile.component.tsx [hunk 4]
```diff
@@ -39,72 +33,86 @@ export default function OrganizationGenerationInformationTile({
...
+                {vcdOrganization?.currentState?.fullName}
+              </Description>
+              <OsdsIcon
+                aria-label="edit"
+                className="mx-6 cursor-pointer"
+                name={ODS_ICON_NAME.PEN}
+                size={ODS_ICON_SIZE.xxs}
+                color={ODS_THEME_COLOR_INTENT.primary}
+                onClick={() => navigate(urlParts.editName)}
+              />
```
Comment (darsene) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-general-information-tile/OrganizationGeneralInformationTile.component.tsx:54]: **niptick** : could be defined as a dedicated component as it's used, a least, twice.

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-options-tile/OptionsTileLicense.tsx (hunk 2 above)
Comment (darsene) [reply packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-options-tile/OptionsTileLicense.tsx:11]: no fix needed here ?

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/billing-tile/BillingTile.component.tsx (hunk 3 above)
Comment (pauldkn) [reply packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/billing-tile/BillingTile.component.tsx]: Yes, it's fixed!

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-options-tile/OptionsTileLicense.tsx (hunk 2 above)
Comment (pauldkn) [reply packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-options-tile/OptionsTileLicense.tsx:11]: Data is now passed using props instead this hook, but I forgot to delete this component (not used anymore), thanks 👍

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/icon/UpdateIcon.component.tsx
Comment (anooparveti) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/components/icon/UpdateIcon.component.tsx:1]: This component has no additional logic as it just displays the OsdsIcon. Avoid creating wrapper components.

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Review by rjamet-ovh — APPROVED** [review]

**Review by darsene — APPROVED** [review]

File: packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-general-information-tile/OrganizationGeneralInformationTile.component.tsx [hunk 5]
```diff
@@ -39,72 +33,86 @@ export default function OrganizationGenerationInformationTile({
...
+            <OsdsText
+              className="mb-4"
+              level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
+              size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+              color={ODS_THEME_COLOR_INTENT.text}
+              hue={ODS_THEME_COLOR_HUE._500}
+            >
+              <RegionLabel
+                code={vcdOrganization?.currentState?.region}
+              ></RegionLabel>
```
Comment (anooparveti) [inline packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-general-information-tile/OrganizationGeneralInformationTile.component.tsx:90]: **nitpick:** We have a new component on `manager-components` from https://github.com/ovh/manager/issues/12226. Can you please plan to remove this component at the µ-app?

**Review by anooparveti — APPROVED** [review]

**Pull request merged**